frontend/src/components/HomePageComponents/LoginButton.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { Button } from 'semantic-ui-react'
+
+const LoginButton = () => (
+    <Button style={{textAlign:'right'}} color={"orange"}>Login</Button>
```

Comment:
Passe le string color directement, pas besoin de l'évaluer inutilement. (et toujours, essaye d'éviter les styles inlines, comme pour ton text align)
```suggestion
    <Button style={{textAlign:'right'}} color="orange">Login</Button>
```